recipes/cyavro/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "cyavro" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+   fn: {{ name }}-{{ version }}.tar.gz
+   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+   sha256: 780c164be6ed4be56f31f7c06ba46c13e3b0968f92824aa636a17e21a07d9e2d
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+  script: python setup.py install
+
+requirements:
+    build:
+        - python
+        - cython
+        - numpy x.x
+        - avro
+        - six
+        - jinja2
+    run:
+        - python
+        - numpy x.x
+        - avro
+        - pandas >=0.16
+        - filelock
+        - six
+        - python-avro
+
+test:
+    requires:
+        - pytest
+        - python-avro
+        - cython
```

Comment:
Is this a test requirement? Would have figured we already used `cython` to compile it so we don't need it any more.